classes/controller/AdminController.php

```diff
@@ -786,7 +786,8 @@ public function processFilter()
         }
 
         $filters = $this->context->cookie->getFamily($prefix.$this->list_id.'Filter_');
-        $definition = ObjectModel::getDefinition($this->className);
+        if (isset($this->className) && $this->className) 
```

Comment:
Thak you for your dedication and help.

https://github.com/PrestaShop/PrestaShop/pull/4195/files

Regards